scamp/scamp-3.c

```diff
@@ -1136,7 +1136,7 @@ void compute_st(void)
         }
     }
 
-    rtr_mc_set(0, 0xffff5555, 0xffffffff, route);
+    rtr_mc_set(0, 0xffff5554, 0xfffffffe, route);
```

Comment:
Magic numbers! Since these appear to be things that have to match with elsewhere in the code, giving them a name would help a lot.

Reply:
There are, of course, other places where the code uses this value. I recommend a header file _in the `scamp` directory_ since it is a value that is only used internally and is not part of any exposed API.

Reply:
I have added these to scamp.h as this is local to scamp only but imported in lots of places.